client_cli/README.md

```diff
@@ -102,3 +102,19 @@ To know how many units of a particular asset an account has, use `asset get` wit
 ```
 
 This query returns the quantity of `XOR#Soramitsu` asset for the `White Rabbit@Soramitsu` account.
+
+### Execute WASM transaction
+
+Use `--file` to specify a path to the WASM file:
+
+```bash
+./iroha_client_cli wasm --file=/path/to/file.wasm
```

Comment:
I think it's `--path`? Also I think it could have been made a positional argument